src/town_gui.cpp

```diff
@@ -1500,16 +1494,18 @@ class HousePickerCallbacks : public PickerCallbacks {
 	int GetSelectedType() const override { return sel_type; }
 	void SetSelectedType(int id) const override { sel_type = id; }
 
+	static HouseZone GetHouseZoneFromClassId(int cls_id) { return HouseZone(to_underlying(HouseZone::TownEdge) + cls_id); }
```

Comment:
Don't we prefer `static_cast` over constructor cast?
```suggestion
	static HouseZone GetHouseZoneFromClassId(int cls_id) { return static_cast<HouseZone>(to_underlying(HouseZone::TownEdge) + cls_id); }
```